docs/release-notes/release-notes-0.16.0.md

```diff
@@ -5,6 +5,10 @@
 * Warning messages from peers are now recognized and
   [logged](https://github.com/lightningnetwork/lnd/pull/6546) by lnd.
 
+* Decrypt onion failure messages with a [length greater than 256
+  bytes](https://github.com/lightningnetwork/lnd/pull/6913). This makes LND
+  more spec compliant.
```

Comment:
s/more/closer to being

Reply:
Good point, spec compliant is a binary thing. Fixed.